src/modules/posts/components/PostProfileView.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { Container, Grid } from '@material-ui/core';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import BorderBottom from '../../../lib/components/Border';
+import LoadingInfo from '../../../lib/components/LoadingInfo';
+import { RootStateType } from '../../../store/rootReducer';
+import { fetchPostById } from '../store/postsSlice';
+import PostInfo from './PostInfo';
+import { useStyles } from '../styles/PostProfileView.styles';
+
+const PostProfileView: React.FC = () => {
```

Comment:
PostProfileView looks line not a good name.. PostProfice - seems like not applicable one

is this component PostView?

Reply:
seems like this is PostView that we discussed